src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v2/SwitchControllerV2.java

```diff
@@ -215,6 +216,34 @@ public CompletableFuture<List<LagPortResponse>> getLagPorts(@PathVariable("switc
         return switchService.getLagPorts(switchId);
     }
 
+    /**
+     * Get LACP status on Switch.
+     *
+     * @param switchId the switch
+     */
+    @ApiOperation(value = "Read all LACP status on specific switch", response = LacpStatusResponse.class)
+    @GetMapping(value = "/{switch_id}/lacpstatus")
+    @ResponseStatus(HttpStatus.OK)
+    public CompletableFuture<List<LacpStatusResponse>> getLacpStatus(@PathVariable("switch_id") SwitchId switchId) {
+        return switchService.getLacpStatus(switchId);
+    }
+
+    /**
+     * Get LACP status.
+     *
+     * @param switchId the switch
+     * @param logicalPortNumber the switch
+     *
+     */
+    @ApiOperation(value = "Read all LACP status on specific switch", response = LacpStatusResponse.class)
```

Comment:
Seems like description should be `Read LACP status on specific port of the switch`
Also, as long as you don't allow ports list or range as an input, you should return `LacpStatusResponse`, not `List`

Reply:
Updated.